src/main/java/seedu/sellsavvy/model/order/Order.java

```diff
@@ -60,6 +60,18 @@ public boolean isSameOrder(Order otherOrder) {
                 && otherOrder.getItem().equals(getItem());
     }
 
+    /**
+     * Returns true if the order has the same item and date as the given {@code Order}
+     * and this order is currently pending.
+     */
+    public boolean isPendingDuplicateOf(Order otherOrder) {
+        if (status != Status.PENDING) {
```

Comment:
is this method going to be reused for editOrder? Have u think of future development where for editOrder edit a already completed order. (I kind of forgot what we discussed on tue already lol :_(.) i thought is we use equal but i can't rmb and i think comparing just item and date and status is fine too.

Reply:
For editOrder, I was planning to re-use this and only execute this check if the new editedOrder has pending status. 
I don't remember what we discussed on tuesday either so i just went with whatever i wrote in the comment under #123 .
Still open to discussion on how we want to implement this however.

Reply:
Changed the implementation to use equals method instead so this method won't be there anymore.